osgp/shared/osgp-ws-smartmetering/src/main/resources/schemas/sm-external-interface.xsd

```diff
@@ -25,6 +25,7 @@
         <xsd:element name="deviceIdentification" type="xsd:string"/>
         <xsd:choice>
           <!-- requests from P4/P5 -->
+          <xsd:element ref="bun:FindEventsRequest"/>
```

Comment:
The change looks OK, but I am surprised to see this XSD in open-smart-grid-platform repository.
If I am not mistaken there isn't any part of the open source GXF code that deals with the external interface which leads me to believe the file is misplaced.

Reply:
@bvdzwet the location of this XSD was discussed with Paul Houtman.